gazebo/msgs/performance_metrics.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto2";
+package gazebo.msgs;
+
+/// \ingroup gazebo_msgs
+/// \interface PerformanceMetrics
+/// \brief A message for run-time simulation performance metrics
+
+message PerformanceMetrics
```

Comment:
We should document this message and all of its fields. [Here](https://github.com/osrf/gazebo/blob/gazebo11/gazebo/msgs/camera_lens.proto)'s an example of a documented proto file.